Milyli.ScriptRunner.Services.Interfaces/V1/IScriptRunManager.cs

```diff
@@ -1,64 +1,66 @@
 ﻿namespace Milyli.ScriptRunner.Services.Interfaces.V1
 {
+	using System;
 	using System.Threading.Tasks;
-	using Models.Requests;
-	using Models.Responses;
+	using Contracts.V1.Requests;
+	using Contracts.V1.Responses;
 	using Relativity.Kepler.Services;
 
 	/// <summary>
 	/// Handles Script Run CRUD operations, as well as History and Execution operations.
 	/// </summary>
 	[WebService("ScriptRunApi")]
 	[ServiceAudience(Audience.Public)]
-	[RoutePrefix("API/V1/ScriptRun")]
+	[RoutePrefix("ScriptRun")]
     public interface IScriptRunManager
 	{
 		/// <summary>
 		/// Read a single Script Run.
 		/// </summary>
-		/// <param name="req">Request defining the single script run to return.</param>
+		/// <param name="scriptRunId">Script run to read.</param>
 		/// <returns>Script Run definition.</returns>
 		[Route("Read")]
-		Task<ReadScriptRunResponse> GetScriptRunAsync(ReadScriptRunRequest req);
+		Task<ScriptRunResponse> ReadSingleAsync(int scriptRunId);
 
 		/// <summary>
 		/// Creates a new Script Run.
 		/// </summary>
 		/// <param name="req">Creation request.</param>
 		/// <returns>Newly created Script Run.</returns>
 		[Route("Create")]
-		Task<CreateScriptRunResponse> CreateScriptRunAsync(CreateScriptRunRequest req);
+		Task<ScriptRunResponse> CreateSingleAsync(ScriptRunRequest req);
 
 		/// <summary>
 		/// Gets the history for a single Script Run.
 		/// </summary>
-		/// <param name="req">History request.</param>
+		/// <param name="scriptRunId">Script Run to read history of.</param>
 		/// <returns>Run History.</returns>
-		[Route("History")]
-		Task<ReadRunHistoryResponse> GetRunHistoryAsync(ReadHistoryRequest req);
+		[Route("ReadHistory")]
+		Task<ReadRunHistoryResponse> GetRunHistoryAsync(int scriptRunId);
 
 		/// <summary>
 		/// Updates definition of a single Script Run.
 		/// </summary>
 		/// <param name="req">Update Request.</param>
 		/// <returns>Updated Script Definition.</returns>
 		[Route("Update")]
-		Task<UpdateScriptRunResponse> UpdateScriptRunAsync(UpdateScriptRunRequest req);
+		Task<ScriptRunResponse> UpdateSingleAsync(ScriptRunRequest req);
 
 		/// <summary>
 		/// Execute a single Script Run on-demand.
 		/// </summary>
-		/// <param name="req">Run Request indicating which script run to execute.</param>
+		/// <param name="scriptRunId">Script Run to execute.</param>
 		/// <returns></returns>
 		[Route("Run")]
-		Task RunAsync(RunScriptRunRequest req);
+		Task RunSingleAsync(int scriptRunId);
```

Comment:
These still need a response value.

- [ ] Return a response (event if it's empty for now)